contrib/python/podman/libs/containers.py

```diff
@@ -93,6 +94,41 @@ def export(self, target):
             results = podman.ExportContainer(self.id, target)
         return results['tarfile']
 
+    def commit(self, image_name, *args, **kwargs):
+        """Create image from container.
+
+        All changes overwrite existing values.
+          See inspect() to obtain current settings.
+
+        Changes:
+            CMD=/usr/bin/zsh
+            ENTRYPOINT=/bin/sh date
+            ENV=TEST=test_containers.TestContainers.test_commit
+            EXPOSE=8888/tcp
+            LABEL=unittest=test_commit
+            USER=bozo:circus
+            VOLUME=/data
+            WORKDIR=/data/application
+        """
+        try:
+            author = kwargs.get('author', getpass.getuser())
+        except Exception:
+            author = ''
+
+        changes = kwargs.get('changes', [])
+        for c in changes:
+            if c.startswith('LABEL') and c.count('=') < 2:
```

Comment:
Should this be `c.startswith('LABEL=')`?

Reply:
Done